.github/workflows/trigger-load-tests.yml

```
@@ -0,0 +1,36 @@
+name: Trigger Load Tests
+
+on:
+  workflow_dispatch:
+    inputs:
+      tests:
+        description: >
+          List of test names
+          This needs to be filled only if you want to run a specific set of tests. Example: [“foo”, “bar”]
+        required: false
+      clusterName:
+        description: 'Cluster name'
+        default: 'bal-perf-cluster-test'
```

```suggestion
        default: 'tcp-perf-cluster-test'
```
Better to change the cluster name

---

Fixed with e28bc2f